packages/typescript-to-proptypes/src/injectPropTypesInFile.ts

```diff
@@ -319,7 +325,7 @@ function plugin(
         if (!node.id) {
           return;
         }
-        const props = propTypes.body.find((prop) => prop.name === node.id!.name);
```

Comment:
I don't know why the return format of the parser was this `ast.body` format, but I find the new one a lot easier to read.